examples/html-kitchen-sink/stories/index.stories.js

```diff
@@ -0,0 +1,26 @@
+import { document } from 'global';
+import { storiesOf } from '@storybook/html';
+import { action } from '@storybook/addon-actions';
+import { linkTo } from '@storybook/addon-links';
+
+import './welcome.css';
+import welcome from './welcome.html';
+
+document.addEventListener('click', e => {
+  const { kind, story } = e.target.dataset;
+  if (kind || story) {
+    e.preventDefault();
+    linkTo(kind, story)();
+  }
+});
+
+storiesOf('Welcome', module).add('Welcome', () => welcome);
+
+storiesOf('Demo', module)
+  .add('heading', () => '<h1>Hello World</h1>')
+  .add('button', () => {
+    const button = document.createElement('button');
+    button.innerText = 'Hello Button';
+    button.addEventListener('click', action('Click'));
```

Comment:
I have some thought on adding some decorator with [backbone-style](http://backbonejs.org/#View-delegateEvents) event listeners:

```
.addDecorator(withActions({
  dblclick : 'Double click',
  'click .icon.doc':   'Doc icon click',
  'contextmenu .icon.doc': 'Doc icon right click'
}))
```

Reply:
Did that: https://github.com/storybooks/storybook/pull/3475/commits/100985bfdbcb5b68adeb63840e1e71086bc14db2